### ISSUE TITLE
Stops sending reports if server answers with status 503. (HS:reports-503)

### ISSUE THREAD

**Issue by lemavri** [prey/prey-node-client#81]
## Problem
Currently, when the client sends the report and gets a 503 response from the server, it stops the generation of further reports. After this, the only way for the client to start sending reports again is for the user to mark the device as recovered and then as missing again, or if the client/device is restarted.

## Log
```
info Tue, 03 Feb 2015 18:19:39 GMT [reports] Report gathered: status
info Tue, 03 Feb 2015 18:19:39 GMT [control-panel] Posting report
info Tue, 03 Feb 2015 18:20:08 GMT [control-panel] Device no longer missing.
warn Tue, 03 Feb 2015 18:20:08 GMT [reports] Canceling stolen report.
info Tue, 03 Feb 2015 18:20:08 GMT [control-panel] Got 503 response: Backend seems to be down.
```

## Suggested solution

A few solutions (not exclusive) might be:
- To check the status of the device the same way the client checks for new instructions.
- Do not stop/disable reports when the server responds with a 503 or alike.
- To retry posting the report at least 3 times, to discard that the 503 response is a temporary issue.

**Comment by fanuneza** [comment]
:+1: to this issue. Thanks for taking care of it, @lemavri.

**Comment by lemavri** [comment]
@fanuneza this has been fixed by 4dea26ada9df825b82a94b88316f6efe2607b596 and will be available in v1.3.7

**Comment by fanuneza** [comment]
This issue is seriously, seriously critical. We have at least four customers with stolen computers that don't report, probably because of this.

This was fixed three weeks ago but still hasn't been deployed :neutral_face:
